packages/dev-server-storybook/src/shared/config/readStorybookConfig.ts

```diff
@@ -60,7 +63,9 @@ export function readStorybookConfig(pluginConfig: StorybookPluginConfig): Storyb
     previewBody = fs.readFileSync(previewBodyPath, 'utf-8');
   }
 
-  const mainJs = validateMainJs(require(mainJsPath));
+  const mainJs = fs.existsSync(commonJsMainPath) ? 
```

Comment:
We are doing `fs.existsSync(commonJsMainPath)` twice. Should we make it a variable?

Reply:
Sure!